### ISSUE TITLE
"Move thread" menu item disapeared in mail 4.0.3 from multi-select menu

### ISSUE THREAD

**Issue by ratermir** [nextcloud/mail#10310]
### Steps to reproduce

After update to NC 30.0.1 and mail app 4.0.3, the menu item "Move thread" disapeared from the menu, opened my click on the "..." button above mail list in case multiple messages are selected. 

### Expected behavior

menu item "Move thread" is in place...

### Actual behavior

![Image](https://github.com/user-attachments/assets/da0d4b0b-798b-47d2-aead-448bef79c174)

### Mail app version

4.0.3

### Nextcloud version

30.0.1

### Mailserver or service

n/a

### Operating system

n/a (frontend)

### PHP engine version

None

### Web server

None

### Database

None

**Comment by GretaD** [comment]
Hello,

Can you please confirm that the screenshot you sent is actually the actual behavior? Because i can see on the screenshot the 6th item is "move 2 threads". You have selected 2 threads and the behavior looks correct.

**Comment by ratermir** [comment]
Sorry, it was my mistake. My language is Czech, due correct translation I switched to English to get the screenshot. 
I am almost sure that in Czech language the item was missing, but  here in the english version it is in place. 

Now, when I switched back to Czech, I can see this menu only in english (and not in chech, how expected), but I am not sure whether it was in chech before and it is not a big issue for me.
So we can close this ticket.
